digital_asset_types/src/dao/scopes/asset.rs

```diff
@@ -290,6 +290,7 @@ pub async fn get_by_id(
         .await?;
     let grouping: Vec<asset_grouping::Model> = asset_grouping::Entity::find()
         .filter(asset_grouping::Column::AssetId.eq(asset.id.clone()))
+        .filter(asset_grouping::Column::GroupValue.is_not_null())
```

Comment:
Need to add filter here: https://github.com/helius-labs/digital-asset-rpc-infrastructure/blob/6a2081dde73e760ca29c5c0bb90160e0e3e54464/digital_asset_types/src/dao/scopes/asset.rs#L291